src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -1,85 +1,266 @@
+        final LineRequest request = new LineRequest(name, color);
+        return postLineRequest(request).header(LOCATION);
+    }
+
+    private ExtractableResponse<Response> requestCreateLine(final String name, final String color) {
+        final LineRequest request = new LineRequest(name, color);
+        return postLineRequest(request);
+    }
+
+    private ExtractableResponse<Response> postLineRequest(final LineRequest request) {
+        return RestAssured.given().log().all()
+            .body(request)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when()
+            .post(BASE_URI)
+            .then().log().all()
+            .extract();
+    }
+
+    private ExtractableResponse<Response> requestGetLines() {
+        return RestAssured.given().log().all()
+            .when()
+            .get(BASE_URI)
+            .then().log().all()
+            .extract();
+    }
+
+    private ExtractableResponse<Response> requestGetLine(final String uri) {
+        return RestAssured.given().log().all()
+            .when()
+            .get(uri)
+            .then().log().all()
+            .extract();
+    }
+
+    private ExtractableResponse<Response> requestUpdateLine(
+        final String name,
+        final String color,
+        final String uri
+    ) {
+        final LineRequest updateRequest = new LineRequest(name, color);
+        return RestAssured.given().log().all()
+            .body(updateRequest)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when()
+            .put(uri)
+            .then().log().all()
+            .extract();
+    }
+
+    private ExtractableResponse<Response> requestDeleteLine(final String uri) {
+        return RestAssured.given().log().all()
+            .when()
+            .delete(uri)
+            .then().log().all()
+            .extract();
+    }
+
+    private void assertCreateLineSuccess(final ExtractableResponse<Response> response) {
+        assertResponseStatusEquals(response, HttpStatus.CREATED);
+        assertThat(response.header(LOCATION)).isNotBlank();
+    }
+
+    private void assertCreateLineFail(final ExtractableResponse<Response> response) {
+        assertResponseStatusEquals(response, HttpStatus.BAD_REQUEST);
+    }
+
+    private void assertGetLinesSuccess(final ExtractableResponse<Response> response) {
+        assertResponseStatusEquals(response, HttpStatus.OK);
+    }
+
+    private void assertGetLinesContained(
+        final ExtractableResponse<Response> response,
+        final List<String> uris
+    ) {
+        final List<Long> expectedLineIds = uris.stream()
+            .map(it -> Long.parseLong(it.split("/")[2]))
+            .collect(Collectors.toList());
+        final List<Long> actualLineIds = response.jsonPath()
+            .getList(".", LineResponse.class)
+            .stream()
+            .map(LineResponse::getId)
+            .collect(Collectors.toList());
```

Comment:
```suggestion
        final List<Long> actualLineIds = response.jsonPath()
            .getList("id", Long.class);
```
`restassured`의 jsonPath를 잘 활용하면 이렇게 간단히도 줄여볼 수 있습니다.
간단하고 명확한 코드는 인수 테스트의 재사용성을 높여주니 참고 부탁드립니다 😊